src/contexts/global.tsx

```diff
@@ -100,6 +108,7 @@ export const GlobalContextProvider: FC<{ overrideContextValue?: PartialDeep<Glob
   const [isOffline, setIsOffline] = useState(false)
   const currentNetwork = getNetworkName(settings.network)
   const newLatestVersion = useLatestGitHubRelease()
+  const [txModalType, setTxModalType] = useState<TxModalType | false>(false)
```

Comment:
```suggestion
  const [txModalType, setTxModalType] = useState<TxModalType>('transfer')
```

Reply:
I think it's safer to set it as false (or null) by default

Reply:
I used `false` as we have similar code in the codebase 😅

Reply:
Ah; in that case you can use `undefined`, which is actually the default which is set by `useState` so you can simplify this to:

```
 const [txModalType, setTxModalType] = useState<TxModalType>()
```

`useState` internally transforms `T` into `T | undefined` so no need to do it yourself; for example, the type internally based on what you wrote originally actually becomes `T | undefined | false`! No need for that